language-configuration.json

```diff
@@ -24,5 +24,7 @@
         ["(", ")"],
         ["\"", "\""],
         ["'", "'"]
-    ]
+    ],
+    // defines what is considered to be a word in the programming language
+    "wordPattern": "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\@\\#\\%\\^\\&\\*\\(\\)\\_\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s]+)"
```

Comment:
What about this? It would still technically match _some_var but it would break those up into two words, so if hoon doesn't even support that then we should be fine.

```suggestion
    "wordPattern": "(-?\\d*\\.\\d\\w*)|(_?[^\\`\\~\\!\\@\\#\\%\\^\\&\\*\\(\\)\\_\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s]+)"
```

Reply:
I took another look at the `hoon.tmLanguage` and realized we could use a simpler pattern that's already used there for arms, auras, etc.

How about this?
```suggestion
    "wordPattern": "[a-z]([a-z0-9-]*[a-z0-9])?"
```